transformations/pig_latin/transformation.py

```diff
@@ -0,0 +1,23 @@
+import piglatin
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+class PigLatin(SentenceOperation):
+    tasks = [
+        TaskType.TEXT_CLASSIFICATION,
+        TaskType.TEXT_TO_TEXT_GENERATION,
+        TaskType.TEXT_TAGGING,
+    ]
+    languages = ["en"]
+
+    def __init__(self, seed=0, max_outputs=1):
+        super().__init__(seed, max_outputs=max_outputs)
+
+    def generate(self, sentence: str):
```

Comment:
It doesn't seem that max_outputs is actually used?

Reply:
Thanks for catching this. Per your suggestion, I added a per-word translation probability. Now that the output is non-deterministic for `0 < replace_prob < 1`, `max_outputs` is actually meaningful, so I went ahead and added functionality to produce multiple outputs.